LICENSE-binary

```diff
@@ -270,27 +288,32 @@ org.apache.zookeeper:zookeeper
 BSD
 ------------
 jline:jline
+com.thoughtworks.paranamer:paranamer
 
 BSD 3-Clause
 ------------
 org.scala-lang:scala-library
 
 Eclipse Distribution License - v 1.0
 ------------------------------------
+jakarta.activation:jakarta.activation-api
 jakarta.xml.bind:jakarta.xml.bind-api
 
 Eclipse Public License (EPL) 2.0
 --------------------------------
-jakarta.servlet:jakarta.servlet-api https://projects.eclipse.org/projects/ee4j.servlet
-
-Common Development and Distribution License (CDDL) 1.1
-------------------------------------------------------
-javax.xml.bind:jaxb-api    https://github.com/javaee/jaxb-v2
+jakarta.annotation:jakarta.annotation-api
+jakarta.servlet:jakarta.servlet-api
+jakarta.ws.rs:jakarta.ws.rs-api
+org.glassfish.hk2:hk2-api
+org.glassfish.hk2:hk2-locator
+org.glassfish.hk2:hk2-utils
+org.glassfish.hk2.external:aopalliance-repackaged
+org.glassfish.hk2.external:jakarta.inject
+org.glassfish.hk2:osgi-resource-locator
 
 MIT License
 -----------
-org.slf4j:slf4j-api
```

Comment:
For binary release, we now bundled

src/main/resources/org/apache/kyuubi/ui/swagger/*
src/main/resources/org/apache/kyuubi/ui/static/assets/fonts/*
src/main/resources/org/apache/kyuubi/ui/static/semantic.min.css
src/main/resources/org/apache/kyuubi/ui/static/semantic.min.js
src/main/resources/org/apache/kyuubi/ui/static/icon.min.css
src/main/resources/org/apache/kyuubi/ui/static/jquery-3.6.0.min.js